src/test/java/posters/flows/OpenHomePageFlow.java

```diff
@@ -18,7 +18,7 @@ public static HomePage flow()
 
         // open home page
         open(Neodymium.configuration().url());
-        HomePage homePage = new HomePage();
+        var homePage = new HomePage();
         homePage.isExpectedPage();
         return homePage;
     };
```

Comment:
please remove the semicolon

Reply:
have a look at the other classes so that this unneeded semicolon can be removed everywhere